### PR TITLE
EZ-496: Article page styles

### PR DESCRIPTION
**Pull request by kb-indrajith**

### CODE REVIEW

File: pattern-lab/source/_patterns/01-molecules/05-HeroComponent/00-hero-component.scss [hunk 1]
```diff
@@ -35,49 +35,52 @@
...
-  cursor: grab;
-  left: 0;
-  bottom: 0;
-  height: auto;
-  padding: 20px 70px 20px 70px !important;
-
-  &.center {
-    text-align: center;
+    min-height: auto;
+    background-color: #ffffff;
```
Comment (kuldev) [inline pattern-lab/source/_patterns/01-molecules/05-HeroComponent/00-hero-component.scss:43]: all below lines can be part of **background** property,
background-color: #ffffff;
background-position: center;
background-repeat: no-repeat;
background-size: 100%;

File: pattern-lab/source/_patterns/01-molecules/05-HeroComponent/00-hero-component.scss [hunk 2]
```diff
@@ -35,49 +35,52 @@
...
+      min-height: 540px;
+      height: 580px;
+    }
+  
+    img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+    div {
```
Comment (kuldev) [inline pattern-lab/source/_patterns/01-molecules/05-HeroComponent/00-hero-component.scss:61]: don't use **div** for typography **div** is for building layouts

File: pattern-lab/source/_patterns/01-molecules/05-HeroComponent/01-hero-carousel.scss [hunk 3]
```diff
@@ -1,9 +1,9 @@
 @import '../../_patterns-global/scss/abstracts/mixins';
 
 .hero_banner_wrapper {
-  height: auto !important;
+  height: auto;
```
Comment (kuldev) [inline pattern-lab/source/_patterns/01-molecules/05-HeroComponent/01-hero-carousel.scss:4]: remove 'height: auto' we don't required this.

File: pattern-lab/source/_patterns/04-pages/01-article-page.scss [hunk 4]
```diff
@@ -27,33 +27,97 @@
...
+        margin: 0 auto;
+      }
+    }
   }
 
   .paragraph--type--gallery {
     max-width: 100%;
     margin: 0 auto;
     @include respond(desktop) {
+      max-width: 960px;
```
Comment (kuldev) [inline pattern-lab/source/_patterns/04-pages/01-article-page.scss:110]: Use the layout values from variables files.

File: pattern-lab/source/_patterns/04-pages/01-article-page.scss [hunk 5]
```diff
@@ -27,33 +27,97 @@
...
+    }
+    @include respond(large-desktop) {
       max-width: 1080px;
```
Comment (kuldev) [inline pattern-lab/source/_patterns/04-pages/01-article-page.scss:113]: Use the layout values from variables files.

File: pattern-lab/source/_patterns/_patterns-global/scss/abstracts/_mixins.scss [hunk 6]
```diff
@@ -66,6 +66,9 @@
   @if $breakpoint == desktop {
     @media only screen and (min-width: 1200px) { @content }; //desktop
   }
+  @if $breakpoint == large-desktop {
+    @media only screen and (min-width: 1300px) { @content }; //desktop
```
Comment (kuldev) [inline pattern-lab/source/_patterns/_patterns-global/scss/abstracts/_mixins.scss:70]: these breakpoints should also be part of variables.

**Review by kuldev — COMMENTED** [review]
@kb-indrajith please check the comments.

**Comment by kuldev** [comment]
@kb-indrajith conflict on file

**Comment by kb-indrajith** [comment]
separate [PR](https://github.com/srijanone/ezcontent_theme/pull/192) created, closing this as duplicate one.

**Pull request closed without merging**